roles/openshift_logging_mux/tasks/main.yaml

```diff
@@ -177,6 +177,48 @@
   check_mode: no
   changed_when: no
 
+# storageclasses are used by default but if static then disable
+# storageclasses with the storageClassName set to "" in pvc.j2
```

Comment:
Im suggesting if we can have a single class where you always pass a value for storageclass and the template has the logic to include or not include the storageclass section based on value passed, that would seem to be preferable to me.  We otherwise have a duplicate task with a singular different value and when conditions